packages/system/kubeovn/Makefile

```diff
@@ -14,23 +14,3 @@ update:
 	patch --no-backup-if-mismatch -p4 < patches/cozyconfig.diff
 	patch --no-backup-if-mismatch -p4 < patches/mtu.diff
 	version=$$(awk '$$1 == "version:" {print $$2}' charts/kube-ovn/Chart.yaml) && \
-	sed -i "s/ARG VERSION=.*/ARG VERSION=$${version}/" images/kubeovn/Dockerfile
-
-image:
-	docker buildx build images/kubeovn \
-		--provenance false \
-		--tag $(REGISTRY)/kubeovn:$(call settag,$(KUBEOVN_TAG)) \
-		--tag $(REGISTRY)/kubeovn:$(call settag,$(KUBEOVN_TAG)-$(TAG)) \
-		--cache-from type=registry,ref=$(REGISTRY)/kubeovn:latest \
-		--cache-to type=inline \
-		--metadata-file images/kubeovn.json \
-		--push=$(PUSH) \
-		--label "org.opencontainers.image.source=https://github.com/cozystack/cozystack" \
-		--load=$(LOAD)
-	REGISTRY="$(REGISTRY)" \
-		yq -i '.global.registry.address = strenv(REGISTRY)' values.yaml
-	REPOSITORY="kubeovn" \
-		yq -i '.global.images.kubeovn.repository = strenv(REPOSITORY)' values.yaml
-	TAG="$(call settag,$(KUBEOVN_TAG))@$$(yq e '."containerimage.digest"' images/kubeovn.json -o json -r)" \
-		yq -i '.global.images.kubeovn.tag = strenv(TAG)' values.yaml
-	rm -f images/kubeovn.json
```

Comment:
Please keep the image target, I hope some day we'll move all images into Cozystack